### ISSUE TITLE
Use exports, not events

### ISSUE THREAD

**Issue by blattersturm** [CritteRo/scalePhone#1]
https://github.com/CritteRo/scalePhone/blob/850b61b711ce5a679cc8cdb8595eecb1b603bc4a/client/cl_phone_events.lua#L3

Unless you're intending to have multiple implementations of 'scalePhone.OpenApp', this should use exports, not events. :P

**Comment by CritteRo** [comment]
That's true. I often use events because they are very easy to "tap into" for whatever reason you might have/need. 

(In this case listening for whenever an app is forcefully opened.)
